src/routes/(docs)/Chip/ExampleChip.svelte

```diff
@@ -0,0 +1,17 @@
+<script lang="ts">
+	import { Chip, Button, Typography } from '$lib';
+</script>
+
+<Chip>
+	<Typography>2 minutes ago</Typography>
```

Comment:
The typography component with no variant doesn't do anything so probably don't need to use it here